zkplus/zkplus.go

```diff
@@ -112,7 +112,7 @@ func (z *ZkPlus) eventLoop() {
 	for {
 		select {
 		case eventToSend = <-whenI(!haveEventToSend && z.connectedChan != nil, z.connectedChan):
-			z.logger.Log(logkey.ZkEvent, eventToSend, logkey.ZkPrefix, z.pathPrefix, "ZkPlus event")
+			z.logger.Log(logkey.ZkEvent, eventToSend, logkey.ZkPrefix, z.pathPrefix, "ZK node modification event")
```

Comment:
add the logkey.message to the string

Reply:
Ack..
May be in the future, we will have to make all logs consistent.

Reply:
let's fix the ones that we want to use others we can do when necessary